packages/vm/src/calls.rs

```diff
@@ -54,6 +56,10 @@ mod read_limits {
     /// Max length (in bytes) of the result data from a ibc_packet_timeout call.
     #[cfg(feature = "stargate")]
     pub const RESULT_IBC_PACKET_TIMEOUT: usize = 64 * MI;
+    /// Max length (in bytes) of the result data from a ibc_source_chain_callback call.
+    pub const RESULT_IBC_SOURCE_CHAIN_CALLBACK: usize = 64 * MI;
+    /// Max length (in bytes) of the result data from a ibc_destination_chain_callback call.
+    pub const RESULT_IBC_DESTINATION_CHAIN_CALLBACK: usize = 64 * MI;
```

Comment:
Independent of this PR: those are generous limits given that the Wasm contract has a memory size limit of 32 MiB. See in wasmd

```
// contractMemoryLimit is the memory limit of each contract execution (in MiB)
// constant value so all nodes run with the same limit.
const contractMemoryLimit = 32
```